src/pages/HowToUse/index.js

```diff
@@ -19,11 +40,47 @@ const HowToUse = () => {
           />
         </Container>
       </Box>
-      <TagGeneratorInstructions />
+      <Box className='containerGray'>
+        <Container>
+          <Grid container className={classes.gridContainer}>
+            <Grid item xs={12} sm={10}>
+              <Typography variant='body1' className={classes.typoStyle}>
+                1. Under your project’s repository, click <SettingsGearIcon />{' '}
+                to paste your tags.
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={10} className={classes.gridPadding}>
```

Comment:
Best practice is to have Grid item columns always add up to 12.

I think it's better to remove `sm={10}` from the top two Grid items. And then to add `item xs={12}` to the bottom two. You could also use nested Box components when the design doesn't really use the columns.

On my computer, the padding and margin of `classes.gridContainer` doesn't seem to be necessary, either.

Reply:
Changed Grid items to nested Box Components in src/pages/HowToUse/index.js 

If I remove classes.gridContainer there is no padding between the blue header and text.

![padding](https://user-images.githubusercontent.com/25930687/130375130-a21d1f54-b55b-4385-923e-3abf767264ad.JPG)